spec/index.html

```diff
@@ -1948,21 +1951,21 @@ <h3>RDF Term Constructors</h3>
-      <tr id="handle-INTEGER"                         ><td style="text-align:left;"            ><a class="type integer"      href="#grammar-production-INTEGER"                         >INTEGER                         </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-literal"      >literal      </a></td><td>The literal has a lexical form of the input string, and a datatype of <code>xsd:integer</code>.</td></tr>
-      <tr id="handle-DECIMAL"                         ><td style="text-align:left;"            ><a class="type decimal"      href="#grammar-production-DECIMAL"                         >DECIMAL                         </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-literal"      >literal      </a></td><td>The literal has a lexical form of the input string, and a datatype of <code>xsd:decimal</code>.</td></tr>
-      <tr id="handle-DOUBLE"                          ><td style="text-align:left;"            ><a class="type double"       href="#grammar-production-DOUBLE"                          >DOUBLE                          </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-literal"      >literal      </a></td><td>The literal has a lexical form of the input string, and a datatype of <code>xsd:double</code>.</td></tr>
-      <tr id="handle-BooleanLiteral"                  ><td style="text-align:left;"            ><a class="type boolean"      href="#grammar-production-BooleanLiteral"                  >BooleanLiteral                  </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-literal"      >literal      </a></td><td>The literal has a lexical form of the <code>true</code> or <code>false</code>, depending on which matched the input, and a datatype of <code>xsd:boolean</code>.</td></tr>
+      <tr id="handle-LANG_DIR"                        ><td style="text-align:left;"            ><a class="type langDir"      href="#grammar-production-LANG_DIR"                        >LANG_DIR                        </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag </a></td><td>The characters following the <a href="#cp-at-sign"><code title="at sign">@</code></a> form the <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a> and optionally the <a data-cite="RDF12-CONCEPTS#dfn-base-direction">initial text direction</a>,<br/>if the matched characters include <a href="#cp-hyphen-hyphen"><code>--</code></a>.<br/>The <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a> MUST be well-formed according to <a data-cite="bcp47#section-2.2.9">section 2.2.9</a> of [[!BCP47]].<br/>If present, the <a data-cite="RDF12-CONCEPTS#dfn-base-direction">initial text direction</a> MUST be either `ltr` or `rtl`.</td></tr>
```

Comment:
This is the only place `<br/>` is used in tabular content, and they're unnecessary. Better to use `<p>` within `<li>` if such sub-blocks are desired or needed.
```suggestion
      <tr id="handle-LANG_DIR"                        ><td style="text-align:left;"            ><a class="type langDir"      href="#grammar-production-LANG_DIR"                        >LANG_DIR                        </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag </a></td><td>The characters following the <a href="#cp-at-sign"><code title="at sign">@</code></a> form the <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a> and optionally the <a data-cite="RDF12-CONCEPTS#dfn-base-direction">initial text direction</a>, if the matched characters include <a href="#cp-hyphen-hyphen"><code>--</code></a>. The <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a> MUST be well-formed according to <a data-cite="bcp47#section-2.2.9">section 2.2.9</a> of [[!BCP47]].<br/>If present, the <a data-cite="RDF12-CONCEPTS#dfn-base-direction">initial text direction</a> MUST be either `ltr` or `rtl`.</td></tr>
```

Reply:
I was not trying to create paragraphs, but control the line-break. Without the `<br/>` the line break made the text less understandable, IMO.

Reply:
I added `style="white-space: nowrap"` to avoid the unfortunate line wrapping of `--`.